src/Coduo/PHPHumanizer/Resources/translations/oxford.en.yml

```diff
@@ -0,0 +1,3 @@
+only_two: "%first% and %second%"
+comma_separated: "%list%, and %last%"
+comma_separated_with_limit: "{1} %list%, and 1 other|[2,Inf] %list%, and %count% others"
```

Comment:
@orestes @lightglitch @dagaa @Forst @sarelvdwalt @Ozmodiar @mattallty @cnkt @tbreuss @IgorDePaula @omissis - anyone willing to help us here with translations?